pkg/api/types.go

```diff
@@ -1946,6 +1946,16 @@ func (p ClusterProfile) IPPoolLeaseType() string {
 	}
 }
 
+// IPPoolLeaseShouldValidateBranch declares whether the ip-pool leases should only be applied to branches matching a
+// specific OpenShift validation model. returns true by default, but should return false for any cluster-profiles
```

Comment:
Ideal condition would be to set it false by default and opt in? (based on only the cloud platforms are using it in this case for now aws)

Reply:
I thought about that, but I think the vast majority of use cases are going to want true. So I made it opt out.